samples/CookieSharing/CookieSharing.WithIdentity.NETFramework/CookieSharing.WithIdentity.NETFramework.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Looks like you have an extra nested directory in here, can you collapse these?

Reply:
They way that had that the VS solution file was in the outer directory and the project file (and the rest of the project's assets) were in the inner folder:

* .vs
* CookieSharing.WithIdentity.NETFramework
   * CookieSharing.WithIdentity.NETFramework.csproj
   * The rest of the project's files
* packages
* CookieSharing.WithIdentity.NETFramework.sln

Reply:
Hrm I guess this is a weird frankenstein situation. is there no way to just use the single AuthSamples.sln and add all projects to that?  And run them each via the same sln?

Reply:
Frankenstein! Yes, we explicitly make our Core samps xplat and IDE-agnostic. ..... well ... truly, we have work to do on that point, but that's the goal.

Reply:
That's what VS does when newing up a proj/sln. I have one here that I did new up a while back, and it did that on that project, too. It likes to land the sln file outside of the project's folder. I presume this is so when projects are added to the solution, the one sln file (outside of the project folders) manages them all.

![capture](https://user-images.githubusercontent.com/1622880/35076820-dfe2a7f0-fbbf-11e7-8f5b-9ad2f8d90349.PNG)

Reply:
Yes I understand that, but we already have a top level AuthSamples.sln, so can we use that instead of creating a new standalone sln.  That should let you arrange it like the other projects with a single folder.

Reply:
Oh, I see what you mean. Not used to the setup, since we're actively getting the last of the VS projects out of the doc samples. I'll fix it up.

Reply:
Yeah repos typically have one top level sln file which include all of the projects.

Reply:
AuthSamples is a bit unique since its meant as a samples repo but its also part of Universe and the CI build so it also needs to behave like a real library repo.